packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics-latencies.ts

```diff
@@ -121,6 +122,21 @@ export default class CallDiagnosticLatencies extends WebexPlugin {
     });
   }
 
+  /**
+   * Accumulate latency for a request
+   * @param callback - callback for which you would like to measure latency
+   * @param key - key
+   * @returns
+   */
+  public accumulateLatency(callback: () => Promise<unknown>, key: PreComputedLatencies) {
+    const start = performance.now();
+
+    return callback().finally(() => {
+      const currentLatency = this.precomputedLatencies.get(key) ?? 0;
+      this.saveLatency(key, currentLatency + (performance.now() - start), true);
+    });
+  }
```

Comment:
we already support this, see saveLatency overwrite param

Reply:
This a special case of the overwrite when we want to add the current value to the next one before overwrite it. 

It is doable with `measureLatency` as well, but we need to access to the current value and I really do not want to use `precomputedLatencies` inside cantina. Actually `precomputedLatencies` should be private.

Reply:
never mind, I just checked again and the saveLatency does what you described.

The `overwrite` name is a bit misleading, because it not just simple overwrite, but also accumulate values

Reply:
measureLatency exposes the overwrite property too, so you don't have to access precomputedLatencies directly in cantina

Reply:
This PR was created for adding downloadIntelligenceModelsReqResp, I strongly feel that changing any util should have been a separate PR so that it could be focused and tested. I will still recommend to create a different PR for this.